lib/nylas/event.rb

```diff
@@ -42,5 +44,15 @@ def rsvp(status, notify_participants:)
                       event_id: id, account_id: account_id)
       rsvp.save
     end
+
+    private
+
+    def query_params
+      return {} if notify_participants.nil?
+
+      {
+        notify_participants: notify_participants
+      }
```

Comment:
Unless that explicit `return` is doing something I'm not following, could this also be

```ruby
def query_params
  if notify_participants.nil?
    {}
  else
    { notify_participants: notify_participants }
  end
end
```
?

Reply:
Fixed here 36d840047b13ce05090484b4d7936e1eb1edca8f